src/emulator/gui/src/controls_dialog.cpp

```diff
@@ -25,7 +25,7 @@ namespace gui {
 
 void draw_controls_dialog(HostState &host) {
     float width = ImGui::GetWindowWidth() / 1.35f;
-    float height = ImGui::GetWindowHeight() / 1.35f;
+    float height = ImGui::GetWindowHeight() / 1.25f;
```

Comment:
Is there any explaination for this

Reply:
i made the list longer when i added the toggle so i had to make the window taller

Reply:
Right